idea/idea-frontend-api/src/org/jetbrains/kotlin/idea/frontend/api/StableNullability.kt

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.idea.frontend.api
+
+enum class StableNullability(val canBeNull: Boolean, val canBeNonNull: Boolean) {
```

Comment:
Please, add KDoc and Kt prefix to `StableNullability`: `KtStableNullability`

Reply:
I will rename it to `KtSmartCastNullability` to distinguish it from `KtTypeNullability` and to make it clear it comes from smart cast. Also removing "stable" from the name but explaining it in the KDoc. I also tried to explain that it is derived from smart cast types.